src/py/flwr/server/strategy/krum.py

```diff
@@ -39,10 +39,40 @@
 from .fedavg import FedAvg
 
 
+# pylint: disable=line-too-long
 class Krum(FedAvg):
-    """Configurable Krum strategy implementation."""
+    """Krum strategy.
```

Comment:
```suggestion
    """Krum [Blanchard et al., 2017] strategy.
    
    Implementation based on https://arxiv.org/abs/1703.02757
```
We could also add the paper link there